clients/admin-ui/src/pages/dataset/new/index.tsx

```diff
@@ -0,0 +1,68 @@
+import {
+  Box,
+  Breadcrumb,
+  BreadcrumbItem,
+  Button,
+  Heading,
+  Stack,
+  Text,
+} from "@fidesui/react";
+import type { NextPage } from "next";
+import NextLink from "next/link";
+import { useState } from "react";
+
+import Layout from "~/features/common/Layout";
+import DatasetYamlForm from "~/features/dataset/DatasetYamlForm";
+
+const NewDataset: NextPage = () => {
+  const [generateMethod, setGenerateMethod] = useState<
+    "yaml" | "database" | "manual" | null
+  >(null);
+  return (
+    <Layout title="Datasets">
+      <Heading mb={2} fontSize="2xl" fontWeight="semibold">
+        Datasets
+      </Heading>
+      <Box mb={8}>
+        <Breadcrumb fontWeight="medium" fontSize="sm" color="gray.600">
+          <BreadcrumbItem>
+            <NextLink href="/dataset">Datasets</NextLink>
+          </BreadcrumbItem>
+          <BreadcrumbItem>
+            <NextLink href="#">Create new dataset</NextLink>
+          </BreadcrumbItem>
+        </Breadcrumb>
+      </Box>
+      <Stack spacing={8}>
+        <Box w={["100%", "100%", "50%"]}>
+          <Text>
+            Choose whether to upload a new dataset YAML, connect to a database
+            using a connection URL or manually generate a dataset.
+          </Text>
+        </Box>
+        <Box>
+          <Button
+            size="sm"
+            mr={2}
+            variant="outline"
+            onClick={() => setGenerateMethod("yaml")}
+            isActive={generateMethod === "yaml"}
+          >
+            Upload a new dataset YAML
```

Comment:
The word "Upload" made me think it there was going to be file-picker. Is the paste-box the main solution for now? If so it's probably worth changing the word. Or if there's additional work planned to do a file-upload then that could use a ticket.